Offseason2022/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -4,27 +4,105 @@
 package frc.robot.subsystems;
 
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+import edu.wpi.first.wpilibj.DataLogManager;
 import edu.wpi.first.wpilibj.PneumaticsModuleType;
 import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.Intake.Mode;
+import frc.robot.frc2135.RobotConfig;
 
 /**
  *
  */
 public class Intake extends SubsystemBase
 {
-  private WPI_TalonFX motorIN8;
-  private Solenoid    arm;
+  private WPI_TalonFX motorIN8      = new WPI_TalonFX(8);
+  private Solenoid    arm           = new Solenoid(0, PneumaticsModuleType.CTREPCM, 0);
+  private boolean     talonValidIN8 = false;
+  private double      acquireSpeed;
+  private double      expelSpeed;
 
   /**
    *
    */
   public Intake( )
   {
-    motorIN8 = new WPI_TalonFX(6);
-
-    arm = new Solenoid(0, PneumaticsModuleType.CTREPCM, 0);
+    setName("Intake");
+    setSubsystem("Intake");
     addChild("Arm", arm);
+
+    SmartDashboard.putBoolean("HL_IN8Valid", talonValidIN8);
+
+    // Check if solenoids are functional or blacklisted
+    if (arm.isDisabled( ))
+      DataLogManager.log(getSubsystem( ) + "Deploy Solenoid is BLACKLISTED");

```

Comment:
Looks good! One suggestion though, you might need to put colons after the getSubsystem( ) calls.